core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -24,8 +24,8 @@ var action = require('./action.js');
 
 var ExplorationEditorSettingsTab = function() {
   /*
-   * Interactive elements
-   */
+  * Interactive elements
+  */
```

Comment:
The comment styles were correct before. Please leave them as-is (here and elsewhere)